2021-09-27 12:35:14,826 INFO    MainThread:18308 [wandb_setup.py:_flush():69] setting env: {}
2021-09-27 12:35:14,826 INFO    MainThread:18308 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-27 12:35:14,827 INFO    MainThread:18308 [wandb_init.py:_log_setup():348] Logging user logs to C:\Users\astelzl\Documents\02_DEV\01_Code_Thesis\NER-prodigy-logWandB\wandb\run-20210927_123514-2vic05fa\logs\debug.log
2021-09-27 12:35:14,827 INFO    MainThread:18308 [wandb_init.py:_log_setup():349] Logging internal logs to C:\Users\astelzl\Documents\02_DEV\01_Code_Thesis\NER-prodigy-logWandB\wandb\run-20210927_123514-2vic05fa\logs\debug-internal.log
2021-09-27 12:35:14,827 INFO    MainThread:18308 [wandb_init.py:_jupyter_setup():300] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001CD2C01CE80>
2021-09-27 12:35:14,829 INFO    MainThread:18308 [wandb_init.py:init():381] calling init triggers
2021-09-27 12:35:14,829 INFO    MainThread:18308 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-09-27 12:35:14,829 INFO    MainThread:18308 [wandb_init.py:init():401] re-initializing run, found existing run on stack: 1fik4rqq
2021-09-27 12:35:14,830 INFO    MainThread:18308 [wandb_run.py:finish():1275] finishing run alfred-ls/Prodigy_NER/1fik4rqq
2021-09-27 12:35:14,831 INFO    MainThread:18308 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-09-27 12:35:14,831 INFO    MainThread:18308 [wandb_run.py:_restore():1547] restore
2021-09-27 12:35:15,176 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 862
}

2021-09-27 12:35:15,448 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 862
}

2021-09-27 12:35:16,302 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 4730
}

2021-09-27 12:35:16,404 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 4759
}

2021-09-27 12:35:16,506 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 4759
}

2021-09-27 12:35:16,607 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4759
  total_bytes: 4759
}

2021-09-27 12:35:16,708 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4759
  total_bytes: 4759
}

2021-09-27 12:35:16,809 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4759
  total_bytes: 4759
}

2021-09-27 12:35:16,910 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4759
  total_bytes: 4759
}

2021-09-27 12:35:17,012 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4759
  total_bytes: 4759
}

2021-09-27 12:35:17,375 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4759
  total_bytes: 4759
}

2021-09-27 12:35:17,835 INFO    MainThread:18308 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4759
  total_bytes: 4759
}
local_info {
}

2021-09-27 12:35:18,902 INFO    MainThread:18308 [wandb_run.py:_show_files():1947] logging synced files
2021-09-27 12:35:18,911 INFO    MainThread:18308 [wandb_init.py:init():430] starting backend
2021-09-27 12:35:18,911 INFO    MainThread:18308 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-09-27 12:35:18,914 INFO    MainThread:18308 [backend.py:ensure_launched():135] starting backend process...
2021-09-27 12:35:18,980 INFO    MainThread:18308 [backend.py:ensure_launched():139] started backend process with pid: 17912
2021-09-27 12:35:18,980 INFO    MainThread:18308 [wandb_init.py:init():435] backend started and connected
2021-09-27 12:35:18,991 INFO    MainThread:18308 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-09-27 12:35:18,991 INFO    MainThread:18308 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-27 12:35:18,991 INFO    MainThread:18308 [wandb_init.py:init():494] updated telemetry
2021-09-27 12:35:18,992 INFO    MainThread:18308 [wandb_init.py:init():517] communicating current version
2021-09-27 12:35:19,800 INFO    MainThread:18308 [wandb_init.py:init():522] got version response 
2021-09-27 12:35:19,800 INFO    MainThread:18308 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-27 12:35:20,076 INFO    MainThread:18308 [wandb_init.py:init():557] starting run threads in backend
2021-09-27 12:35:20,879 INFO    MainThread:18308 [wandb_run.py:_console_start():1605] atexit reg
2021-09-27 12:35:20,879 INFO    MainThread:18308 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-09-27 12:35:20,879 INFO    MainThread:18308 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-09-27 12:35:20,880 INFO    MainThread:18308 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-27 12:35:20,881 INFO    MainThread:18308 [wandb_init.py:init():582] run started, returning control to user process
2021-09-27 12:35:23,069 INFO    MainThread:18308 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-27 12:35:23,070 INFO    MainThread:18308 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-27 12:37:09,521 INFO    MainThread:18308 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-27 12:37:09,949 INFO    MainThread:18308 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-27 12:37:09,949 INFO    MainThread:18308 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-27 12:37:31,257 INFO    MainThread:18308 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-27 12:37:31,259 INFO    MainThread:18308 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-27 12:37:31,259 INFO    MainThread:18308 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-27 13:37:34,182 INFO    MainThread:18308 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-27 13:37:34,202 INFO    MainThread:18308 [wandb_setup.py:_flush():69] setting env: {}
2021-09-27 13:37:34,203 INFO    MainThread:18308 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-27 13:37:34,203 INFO    MainThread:18308 [wandb_init.py:_log_setup():348] Logging user logs to C:\Users\astelzl\Documents\02_DEV\01_Code_Thesis\NER-prodigy-logWandB\wandb\run-20210927_133734-124nv1i7\logs\debug.log
2021-09-27 13:37:34,203 INFO    MainThread:18308 [wandb_init.py:_log_setup():349] Logging internal logs to C:\Users\astelzl\Documents\02_DEV\01_Code_Thesis\NER-prodigy-logWandB\wandb\run-20210927_133734-124nv1i7\logs\debug-internal.log
2021-09-27 13:37:34,204 INFO    MainThread:18308 [wandb_init.py:init():381] calling init triggers
2021-09-27 13:37:34,204 INFO    MainThread:18308 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-09-27 13:37:34,204 INFO    MainThread:18308 [wandb_init.py:init():401] re-initializing run, found existing run on stack: 2vic05fa
2021-09-27 13:37:34,206 INFO    MainThread:18308 [wandb_run.py:finish():1275] finishing run alfred-ls/Prodigy_NER/2vic05fa
